frontend/src/common/ConfirmationDialogue.tsx

```diff
@@ -24,7 +36,7 @@ export default function ConfirmationDialogue({
   dialogMessage = 'Are you sure you want to perform this action?',
 }: ConfirmationDialogProps) {
   return (
-    <Dialog fullWidth open={open} onClose={onCancel}>
+    <Dialog fullWidth open={open} onClose={onCancel} keepMounted disableEscapeKeyDown TransitionComponent={Transition}>
```

Comment:
We should ensure that we are WCAG 2.0 AA compliant, which requires the ability to use common key combos to escape dialogs.

Reply:
I have removed this to be compliant